bin/hoc-2015-nov-9-mailing/generate-all-mailing-lists

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
```

Comment:
No. It's taken care of in generate. See https://github.com/code-dot-org/code-dot-org/pull/5241/files#diff-cbd28d9350bdb535598d0a71190936caR175